v2/cn/blog-understand-cosocket.md

```diff
@@ -0,0 +1,82 @@
+<!---
+    @title         怎样理解 cosocket
+    @creator       覃冠日，李张勇
+    @created       2016-08-01 17:24 GMT
+    @modifier      覃冠日，李张勇
```

Comment:
Please remove this field so that the toolchain can automatically use the git history.